mobile/test/common/integration/BUILD

```diff
@@ -41,6 +41,25 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
+    name = "cds_integration_test",
+    srcs = ["cds_integration_test.cc"],
+    exec_properties = {
+        # TODO(willengflow): Remove this once the sandboxNetwork=off works for ipv4 localhost addresses.
```

Comment:
@WillEngFlow does this still apply?

Reply:
This comment is repeated many times throughout the codebase fwiw.  Whenever this particular exec_property is used